whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/ElasticFullTextSearchTestIntegration.java

```diff
@@ -223,7 +223,8 @@ public void search_multiple_results_with_facet() {
     }
 
     @Test
-    public void search_multiple_results_with_search_limit() {
+    public void search_multiple_results_with_first_three_records() {
+
```

Comment:
Can you add a testcase where personal accounting is increased when we did a fulltext search?

Reply:
not easy :S This is why I think that we don't have that test case already
I have already tried and I could perform that test but I need to change the constant of max people to a constant

Reply:
I tried using it but this works when you perform the search before processing the ES output. I have already debugged that. But yes, lets talk separately

Reply:
ok, done as discussed via Mattermost. I am adding another test for the counting error that I said you